graphql/mutation_resolver.go

```diff
@@ -178,34 +177,28 @@ func (r *mutationResolver) CreateDistro(ctx context.Context, opts CreateDistroIn
 // SaveDistroSection is the resolver for the saveDistroSection field.
```

Comment:
Can we add to this comment to indicate that we do input the entire distro object as opposed to just the individual section?